fdbmonitor/fdbmonitor.cpp

```diff
@@ -426,7 +426,7 @@ struct Command {
 
 		char* endptr;
 		const char* rd =
-		    get_value_multi(ini, "restart_delay", ssection.c_str(), section.c_str(), "general", "fdbmonitor", nullptr);
+		    get_value_multi(ini, "restart-delay", ssection.c_str(), section.c_str(), "general", "fdbmonitor", nullptr);
```

Comment:
For backward compatibility, I guess we also need fdbmonitor/FDBService to be able to accept underscores and hyphens interchangeably.

Reply:
Thanks. I updated the code in the fdbmonitor and FDBService. Is there any other files we need to worry about the compatibility?